include/SFML/Network/Socket.hpp

```diff
@@ -62,10 +62,7 @@ class SFML_NETWORK_API Socket
     /// \brief Some special values used by sockets
     ///
     ////////////////////////////////////////////////////////////
-    enum
-    {
-        AnyPort = 0 //!< Special value that tells the system to pick any available port
-    };
+    static constexpr int AnyPort{0}; // NOLINT(readability-identifier-naming)
```

Comment:
This should be `unsigned short` because it is the type we use to represent port in the network module.